vendor/github.com/gorilla/mux/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+# gazelle:ignore
```

Comment:
just curious, why doesn't this work?

Reply:
There's a bug in the way gazelle handles go build tags. That package has two implementations of a feature inside it, one is used with go versions < 1.7, the other with >= 1.7. Gazelle doesn't understand that and breaks :(

Reply:
ahh, makes sense.